tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -24,6 +39,70 @@ DRAW_TEST_F(CGContext, RedBox, UIKitMimicTest) {
     CGContextFillRect(context, CGRectInset(bounds, 10, 10));
 }
 
+DRAW_TEST_F(CGContext, DrawIntoRect, UIKitMimicTest) {
+    // Create a context to draw the Image into
```

Comment:
please DISABLED_ your new tests for now; we can force-run them with GTestRunner in the interim.